de.js

```diff
@@ -693,55 +693,55 @@ social.trading.submitting Angebot einreichen...
+account.delete.cancel Bist du dir sicher das du die Beantragung zur Accountlöschung zurückziehen willst?
+spin.free.available GRATIS KR VERFÜGBAR!
+spin.free.claim Du kannst diesen spin jede Stunde nach absolvieren eines matchs einlösen.
+spin.free.disable Diese Popups können in den generellen Einstellungen ausgeschalten werden.
+mod.reset.0 Respawne um Effekt zu nehmen
+mod.reset.1 Karten Objekte werden nicht zurückgesetzt
+windows.profile.picture Bild ändern
 stats.assists Assists
 stats.nukes Nukes
-stats.melee Melee
-stats.fistkills Beatdowns
-stats.thrownkills Bullseyes
-stats.headshots Headshots
+stats.melee Nahkampf
+stats.fistkills Niedergeschlagen
+stats.thrownkills Wurfmesser
+stats.headshots Kopfschüsse
 stats.wallbangs Wallbangs
-stats.sprays Sprays Placed
+stats.sprays Graffiti platziert
 windows.host.default.settings Enabling Default Settings allows for storing Player Progress and Highscores
-windows.host.default.commands *Host Commands are Disabled in this mode: /noclip /god /smite etc
-windows.host.rewards Enabling KR Rewards forces the Server to use Default Settings & allows players to earn KR from the map funds:
-windows.store.wheels.sub Try your luck and spin for a chance to win some new skins!
-windows.store.currency * Prices in USD can be converted to your local currency at an exchange rate defined by the payment provider or bank
-windows.store.creator Consider Support your favorite Content Creator by using a Creator Code
-windows.chall.customs Challenges can't be completed in Custom Matches
-twitch.link Are you sure you want to link a twitch account?
-twitch.unlink Are you sure you want to unlink your twitch account?
-windows.clan.leave Are you sure you want to leave the clan?
-windows.clan.delete Are you sure you want to delete the clan?
-windows.clan.kick Are you sure you want kick {0} from the clan?
-windows.store.premium.purchase Are you sure you want to purchase {0} days of Premium?
-windows.playerlist.ban Are you sure you want ban {0} from this game?
-windows.playerlist.flag Are you sure you want to Flag {0}?
-windows.playerlist.unflag Are you sure you want to Unflag {0}?
-windows.playerlist.punish Are you sure you want to take action on {0}?
+windows.host.default.commands *Host Befehle sind in diesem Modus deaktiviert: /noclip /god /smite etc
+windows.host.rewards KR Belohnungen einzuschalten zwingt den Server Standart Einstellungen zu verwenden und erlaubt Spielern KR von der Map Kasse zu verdienen
+windows.store.wheels.sub Versuche dein Glück und dreh den Spin für die Chance auf coole Skins
+windows.store.currency * Preise in USD können zu deiner Lokalen Währung umgewandelt werden. Diese Währungsumrechnung ist von deiner Zahlungsanbieter oder Bank bestimmt.
+windows.store.creator Überleg dir einen Crator Code zu benutzem um deinen lieblingscreator zu unterstützen
+windows.chall.customs Herausforderungen können nicht in privaten Spielen gemacht werden

```

Comment:
Herausforderungen können nicht in Spielen erledigt werden, die nicht offiziell sind.